core/schema/src/main/liquibase/26.0.0/changelog.xml

```diff
@@ -88,4 +88,22 @@
                                  referencedTableName="graph_elements" referencedColumnNames="id" onDelete="CASCADE"/>
     </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+    <changeSet author="pschweizer" id="26.0.0-create-timeseries_meta-table">
+        <createTable tableName="timeseries_meta">
+            <column name="resourceid" type="text">
```

Comment:
Can we create an index on the `resourceid` column?

Reply:
done.